### PR TITLE
Build and insert start animation

### PR DESCRIPTION
**Pull request by Susanne1307**
Screen-Recording-2020-04-12-at-1

### CODE REVIEW

File: client/src/components/StartAnimation.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import leo1 from '../assets/leo1.svg';
+import leo2 from '../assets/leo2.svg';
+import { Foos, Buddy } from '../components/Title';
+
+export const AnimationContainer = styled.div`
+  width: 100%;
```
Comment (phlgr) [inline client/src/components/StartAnimation.js:8]: Maybe change these to `100vw` and `100vh` respectively.

File: client/src/App.js [hunk 2]
```diff
@@ -1,21 +1,8 @@
 import React from 'react';
-import logo from '../src/assets/app_logo.svg';
-import './App.css';
+import StartAnimation from '../src/components/StartAnimation';
 
 function App() {
-  return (
-    <div className="App">
```
Comment (phlgr) [inline client/src/App.js:7]: Maybe do a separate Pull-Request next time for removing the starter code.

File: client/src/assets/leo1.svg [hunk 3]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (phlgr) [inline client/src/assets/leo1.svg:1]: Try minifying this svg.

File: client/src/assets/leo2.svg [hunk 4]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (phlgr) [inline client/src/assets/leo2.svg:1]: Same as other svg file.

File: client/src/stories/0-Animation.stories.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import StartAnimation from '../components/StartAnimation';
+
+export default {
+  title: 'Animation',
```
Comment (phlgr) [inline client/src/stories/0-Animation.stories.js:5]: Add what component you are using with `title: 'Animation',
  component: StartAnimation,`

File: client/src/components/StartAnimation.js [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+  height: 100%;
+  padding-top: 100px;
+`;
+
+export const StartAnimationLeft = styled.div`
```
Comment (phlgr) [inline client/src/components/StartAnimation.js:13]: Excellent animation. Great Job 👍🏻

File: client/src/assets/leo1.svg (hunk 3 above)
Comment (Susanne1307) [reply client/src/assets/leo1.svg:1]: Thanks for the advice, I turned it into an issue (#14 ) 🔖

File: client/src/components/StartAnimation.js (hunk 1 above)
Comment (Susanne1307) [reply client/src/components/StartAnimation.js:8]: I tried it before with vh, but it did not take up the whole screen.

File: client/src/components/StartAnimation.js (hunk 6 above)
Comment (Susanne1307) [reply client/src/components/StartAnimation.js:13]: Thanks 🙂

**Pull request merged**